### PR TITLE
add jenkins e2e suite to play nice with openshift-tests

### PR DESCRIPTION
**Pull request by gabemontero**
/assign @smarterclayton 
/assign @bparees

### CODE REVIEW

File: cmd/openshift-tests/e2e.go [hunk 1]
```diff
@@ -84,6 +84,17 @@ var staticSuites = []*ginkgo.TestSuite{
 		Parallelism: 7,
 		TestTimeout: 20 * time.Minute,
 	},
+	{
+		Name: "openshift/jenkins-e2e",
+		Description: templates.LongDesc(`
+		Tests that exercise the OpensShift / Jenkins integrations provided by the OpenShift Jenkins image/plugins and the Pipeline Build Strategy.
+		`),
+		Matches: func(name string) bool {
+			return strings.Contains(name, "openshift pipeline")
```
Comment (bparees) [inline cmd/openshift-tests/e2e.go]: time for a [jenkins] or [pipeline] tag on the relevant extended tests?

**Comment by bparees** [comment]
/lgtm
but consider introducing a [] tag in the future

File: cmd/openshift-tests/e2e.go (hunk 1 above)
Comment (gabemontero) [reply cmd/openshift-tests/e2e.go]: only if you make me

**Comment by gabemontero** [comment]
thanks and 10-4 ^^

**Pull request merged**